libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/verification/SessionVerificationService.kt

```diff
@@ -39,7 +39,7 @@ interface SessionVerificationService {
     val sessionVerifiedStatus: StateFlow<SessionVerifiedStatus>
 
     /**
-     * Returns whether the current session needs to be verified and the SDK is ready to start the verification.
+     * Returns whether the current session needs to be verified.
```

Comment:
Maybe take the opportunity to rename `canVerifySessionFlow` to `mustVerifySessionFlow` or `needToVerifySessionFlow` for clarity?

Also `SessionVerificationService.isReady` is not used externally, so maybe remove it from the interface?